src/Application/Folders/Commands/AddFolder/AddFolderCommandValidator.cs

```diff
@@ -0,0 +1,26 @@
+using FluentValidation;
+
+namespace Application.Folders.Commands.AddFolder;
+
+public class AddFolderCommandValidator : AbstractValidator<AddFolderCommand>
+{
+    public AddFolderCommandValidator()
+    {
+
```

Comment:
spacing?

Reply:
fixed at https://github.com/ProFile-org/ProFile-BE/pull/80/commits/d87261e746bf0889e0651581bc05148e31e0eef9